synapse/rest/__init__.py

```diff
@@ -54,14 +72,21 @@ def __init__(self, hs):
     def register_servlets(client_resource, hs):
         versions.register_servlets(client_resource)
 
-        # "v1"
+        if not PY3:
+            # "v1" (Python 2 only)
+            v1_register.register_servlets(hs, client_resource)
+
+            # Deprecated in r0
+            events.register_servlets(hs, client_resource)
```

Comment:
this is duplicated below, and probably can't actually be made PY2-only, because it provides https://matrix.org/docs/spec/client_server/unstable.html#id95, which isn't deprecated

Reply:
Hmm, yeah. Fixed.